pkg/controller/controllercmd/cmd.go

```diff
@@ -90,9 +90,6 @@ func (c *ControllerCommandConfig) NewCommandWithContext(ctx context.Context) *co
 
 			defer logs.FlushLogs()
 			defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"), c.version)()
-			defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
-
-			serviceability.StartProfiler()
```

Comment:
why did you kill our profiler?

Reply:
there is already profiler in apiserver :-) so in fact we running 2 profilers today